db/seeds.rb

```diff
@@ -1,4 +1,4 @@
-inning = Inning.create!(name: "2203", current: true, start_date: Date.today + 3.weeks)
+inning = Inning.create!(name: "2203", current: true, start_date: Date.today)
```

Comment:
Perhaps a dumb question, so feel free to ignore/disregard, but is it worth doing a calculation here for the actual current Mod number at the time, based on `Date.today`? I wonder if we could take `Date.today` and figure out that it needs to be `2311`, using "`YYMM`" or something like that, if that makes sense).

Reply:
This is a fun and quick fix, the next push will accommodate for this.